cmd/monaco/main.go

```diff
@@ -69,6 +69,8 @@ We can't wait for your feedback.
 
 	app := cli.NewApp()
 
+	app.ArgsUsage = "[context path]"
```

Comment:
What's the context path? :thinking: The working directory?

Reply:
Good point ^^